support-frontend/assets/helpers/checkouts.js

```diff
@@ -120,7 +123,8 @@ function getPaymentMethodToSelect(
 
 function getPaymentMethodFromSession(): ?PaymentMethod {
   const pm: ?string = storage.getSession('selectedPaymentMethod');
-  if (pm === 'DirectDebit' || pm === 'Stripe' || pm === 'PayPal') {
+  // can't use Flow types for these comparisons for some strange reason
```

Comment:
There is a Flow pattern for this sort of thing. But you'd need to change `assets/helpers/paymentMethods.js` and everywhere that uses it so probably not for this PR. https://flow.org/en/docs/types/utilities/#toc-keys

```javascript
const paymentMethods = {
  Stripe: 'Stripe',
  PayPal: 'PayPal',
  ...
}

type PaymentMethod = $Keys<typeof paymentMethods>;
```

so then you get the union type, and a runtime value so you can do:

```javascript
if (Object.keys(paymentMethods).includes(pm)) {
  ...
}
```